### PR TITLE
TobyBot: check palindromes in your Github issues!

### PR DESCRIPTION
**Pull request by imrehg**
Adding new demo bot.

### CODE REVIEW

File: lib/bots/tobybot.ts [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+        if (palindromeCount === 1) {
+          countWord = 'palindrome';
+        }
+
+        // Test if the whole issue is a palindrome!
+        let palindromeCombo: boolean = false;
+        const bodySimplified = body.replace(/\W/g, '').toLowerCase();
+        if ( bodySimplified === bodySimplified.split('').reverse().join('') ) {
+          palindromeCombo = true;
+        }
```
Comment (lurch) [inline lib/bots/tobybot.ts]: ...which would then simplify this to just `const palindromeCombo = isPalidrome(body.replace(/\W/g, ''));`

File: lib/bots/tobybot.ts [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+ */
+
+// TobyBot listens for Github Issues, and counts the palindromes in them
+//
+// To set up, follow the main guide, and define these respective env vars:
+// TOBYBOT_WEBHOOK_SECRET
+// TOBYBOT_INTEGRATION_ID
+// TOBYBOT_PEM
+// TOBYBOT_NAME
+// TOBYBOT_EMAIL
```
Comment (imrehg) [inline lib/bots/tobybot.ts]: @hedss I guess that's a holdover from the syncbot example and the email setting is not really needed, right? Or I'm double-missing something?

Comment (hedss) [inline lib/bots/tobybot.ts]: Sounds like it, yes! I don't see any reason to need it.

**Comment by hedss** [comment]
Currently only holding off reviewing this as VersionBot needs a change to allow it to see forked repos that aren't owned by Resin. Once that work is in, we'll come back to this!

**Comment by imrehg** [comment]
@hedss thanks, noted, and will address the other comments before before continuing too.

**Comment by imrehg** [comment]
@hedss how are bots, would this PR be something that can be revived, or should be dropped? Just as a current status report, I know that there are more important things to do, just to see whether I should keep this around for the future. And not a hurry for any reply. :)

**Comment by hedss** [comment]
@imrehg I don't want to drop it, but it requires the changes to allow non-resin org owned repos. I'm hoping I'll get some time to do this, or someone else will, early next year!

**Comment by lurch** [comment]
Have a safe Christmas, TobyBot :gift: :christmas_tree: :gift: toByboT ,samtsirhC efas a evaH

**Pull request closed without merging**